frontend/src/services/hardwareProfileService.ts

```diff
@@ -0,0 +1,38 @@
+import axios from '~/utilities/axios';
+import { HardwareProfileKind } from '~/k8sTypes';
+import { ResponseStatus } from '~/types';
+
+export const createHardwareProfile = (
```

Comment:
we dont want to do backend stuff, we should expect it to be available where we do the pass through calls for list and get. just create all the delete, post, and update calls there 

like this and so on

```tsx
export const deleteHardwareProfile = (name: string, namespace: string): Promise<HardwareProfileKind> =>
  k8sDeleteResource<HardwareProfileKind>({
    model: HardwareProfileModel,
    queryOptions: { name, ns: namespace },
  });
```